docs/pages/Scope.md

```diff
@@ -0,0 +1,35 @@
+## The Scope of Adoptium Quality Assurance (AQA)
+
+The Eclipse AQAvit project consists of several Github repositories and the main objective of the project is to deliver a quality assurance program that can be utilized in several ways.  This quality assurance program called Adoptium Quality Assurance (AQA) ensures the reliability and excellence of OpenJDK builds distributed by the Eclipse Temurin project and is also used by many other OpenJDK distributors.
+
+Let's explore the different ways users can leverage the work being done at the AQAvit project:
+
+### 1. AQAvit Verification:
+AQAvit is primarily used for verification testing of the OpenJDK builds. This application involves verifying an OpenJDK build against a specific version of the test material.  All tests must be shown to pass and the results of this automated testing must be shared publicly along with transparent reporting of any test exclusions.  The automated tests cover a range of quality criteria including:
+
+- Functional Correctness: - AQAvit tests the functional correctness of the OpenJDK builds, ensuring that they execute Java applications and libraries as expected and according to the Java SE standard.
+
+- Stability and Reliability: - AQAvit assesses the stability and reliability of the OpenJDK builds, verifying that they operate without crashes or unexpected behavior under various conditions.
+
+- Performance Benchmarking: - AQAvit includes performance tests to benchmark the OpenJDK builds, ensuring that they meet acceptable performance criteria for a wide range of applications.
+
+- Security Testing: AQAvit verifies OpenJDK builds against several security test suites and for known vulnerabilities.
+
+AQAvit verification is focussed on testing for 'quality'.  Compatibility testing is done as part of the [Eclipse Temurin Compliance project](https://projects.eclipse.org/projects/adoptium.temurin-compliance) and is outside of the scope of AQAvit.
```

Comment:
I think we need to reference this page: https://adoptium.net/docs/aqavit-verification/ for what AQAvit Verification is (and means to consumers).